README.md

```diff
@@ -149,28 +149,29 @@ If your `package.json` does not have `"type": "module"`, use `.mjs` for the exte
+| Set the inspector host to listen on (default: loopback address or `0.0.0.0` inside Docker or Kubernetes)                                                                         |               | `--debug-host`              | `FASTIFY_DEBUG_HOST`              |
+| Prints pretty logs                                                                                                                                                               | `-P`          | `--pretty-logs`             | `FASTIFY_PRETTY_LOGS`             |
+| Watch process.cwd() directory for changes, recursively; when that happens, the process will auto reload                                                                          | `-w`          | `--watch`                   | `FASTIFY_WATCH`                   |
+| Ignore changes to the specified files or directories when watch is enabled. (e.g. `--ignore-watch='node_modules .git logs/error.log'` )                                          |               | `--ignore-watch`            | `FASTIFY_IGNORE_WATCH`            |
+| Prints events triggered by watch listener (useful to debug unexpected reload when using `--watch` )                                                                              | `-V`          | `--verbose-watch`           | `FASTIFY_VERBOSE_WATCH`           |
+| Use custom options                                                                                                                                                               | `-o`          | `--options`                 | `FASTIFY_OPTIONS`                 |
+| Set the prefix                                                                                                                                                                   | `-x`          | `--prefix`                  | `FASTIFY_PREFIX`                  |
+| Set the plugin timeout                                                                                                                                                           | `-T`          | `--plugin-timeout`          | `FASTIFY_PLUGIN_TIMEOUT`          |
+| Set forceCloseConnections option on fastify instance                                                                                                                             | `-f`          | `--force-close-connections` | `FASTIFY_FORCE_CLOSE_CONNECTIONS` |
+| Defines the maximum payload, in bytes,<br>that the server is allowed to accept                                                                                                   |               | `--body-limit`              | `FASTIFY_BODY_LIMIT`              |
+| Set the maximum ms delay before forcefully closing pending requests after receiving SIGTERM or SIGINT signals; and uncaughtException or unhandledRejection errors (default: 500) | `-g`          | `--close-grace-delay`       | `FASTIFY_CLOSE_GRACE_DELAY`       |
```

Comment:
Why did you reformat the whole table? Can you only make the change needed?

Reply:
Ops, I believed was some linter. Instead was my IDE. Restored the untouched lines